pages/domains/domains_election.md

```diff
@@ -12,49 +12,51 @@ eleventyNavigation:
+Though .gov domain registration and renewal are free, there are often costs associated with moving to a new domain. These costs may include hiring technical staff or consultants to facilitate the switch, replacing printed materials, and informing the public of the change.
 
 [View our checklist for moving to .gov](../moving/).
 
 While we cannot guarantee access to funds, election offices may wish to seek funding from the following sources.
 
 #### State and Local Cybersecurity Grant Program 
 
-The [DHS State and Local Cybersecurity Grant program](https://www.cisa.gov/state-and-local-cybersecurity-grant-program) cites ["migration to the .gov internet domain"](https://www.cisa.gov/state-and-local-cybersecurity-grant-program-frequently-asked-questions) as a cybersecurity best practice that must be included as part of an application's Cybersecurity Plan. Eligible entities can use grant funds to implement their Cybersecurity Plan, which includes costs associated with moving to .gov.
+The [DHS State and Local Cybersecurity Grant program](https://www.cisa.gov/state-and-local-cybersecurity-grant-program) cites [“migration to the .gov internet domain”](https://www.cisa.gov/state-and-local-cybersecurity-grant-program-frequently-asked-questions) as a cybersecurity best practice that must be included as part of an application’s Cybersecurity Plan. Eligible entities can use grant funds to implement their Cybersecurity Plan, which includes costs associated with moving to .gov.
 
 #### Homeland Security Grant Program
 
-The [DOTGOV Act](https://www.congress.gov/bill/116th-congress/house-bill/133/text/enr){.usa-link--external} made "migrating any online service" to .gov an allowable expense under the [Homeland Security Grant Program](https://www.fema.gov/grants/preparedness/homeland-security){.usa-link--external}.  FEMA manages the grant program, and potential grantees may include transition costs in their investment justification. For more information, read [FEMA's preparedness grants manual](https://www.fema.gov/sites/default/files/documents/FEMA_2021-Preparedness-Grants-Manual_02-19-2021.pdf#page=50){.usa-link--external} (PDF, page 50).
+The [DOTGOV Act](https://uscode.house.gov/statviewer.htm?volume=134&page=2298#){.usa-link--external} made “migrating any online service” to .gov an allowable expense under the [Homeland Security Grant Program](https://www.fema.gov/grants/preparedness/homeland-security){.usa-link--external}.  FEMA manages the grant program, and potential grantees may include transition costs in their investment justification. For more information, read [FEMA’s preparedness grants manual](https://www.fema.gov/sites/default/files/documents/fema_gpd-fy-23-preparedness-grants-manual.pdf#page=55){.usa-link--external} (PDF, page 55).
```

Comment:
The Google Doc uses this link instead for the "DOTGOV Act": https://www.congress.gov/bill/116th-congress/house-bill/133/text/enr

Reply:
We should use the uscode.house.gov link.

Reply:
Yes, the UI has the correct link - I didn't update it properly in the Google doc. I just fixed that. Sorry for the confusion!